### DISCUSSION TITLE
export monthvalues

### DISCUSSION THREAD

**Discussion by SimonAM** [js-fns/date-fns#2874] [Q&A]
Is there a way of extracting the date-fns locale data such as 
 - monthValues.wide
 - monthValues.abbreviated
 - dayValues.wide
 - dayValues.abbreviated
 
 
**My use case**
Im using the calendar library [React native calendars](https://github.com/wix/react-native-calendars) that uses XDate for i18n. 
XDate has a [more simple locale object](https://gist.github.com/arshaw/1221376) than date-fns but does not provide the language data out of the box.

**Comment by fturmel** [comment]
You can access these through the `.localize.day()` and `.localize.month()` functions of each locale, and potentially build and cache these values in your app.

```js
import { frCA } from "date-fns/locale";

const locale = frCA;

const days = [0, 1, 2, 3, 4, 5, 6].map((i) => {
  return {
    wide: locale.localize.day(i, { width: "wide" }),
    abbreviated: locale.localize.day(i, { width: "abbreviated" })
  };
});

const months = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11].map((i) => {
  return {
    wide: locale.localize.month(i, { width: "wide" }),
    abbreviated: locale.localize.month(i, { width: "abbreviated" })
  };
});

console.log(days);
console.log(months);

```

<img width="525" alt="Screen Shot 2021-12-24 at 11 02 31 AM" src="https://user-images.githubusercontent.com/4009209/147363565-7dfd74bc-4525-4cd3-a230-ee6413873514.png">
